crates/bevy_animation/src/lib.rs

```diff
@@ -639,106 +676,102 @@ fn apply_animation(
     parents: &Query<(Has<AnimationPlayer>, Option<&Parent>)>,
     children: &Query<&Children>,
 ) {
-    if let Some(animation_clip) = animations.get(&animation.animation_clip) {
-        // We don't return early because seek_to() may have been called on the animation player.
-        animation.update(
-            if paused { 0.0 } else { time.delta_seconds() },
-            animation_clip.duration,
-        );
+    let Some(animation_clip) = animations.get(&animation.animation_clip) else {
```

Comment:
This `let else` block saves us indentation across the entire method: hence the large diff. Commit-wise reviewing may be easier.

Reply:
can also use "hide whitespace" in github